src/config.c

```diff
@@ -3373,6 +3373,9 @@ standardConfig static_configs[] = {
     createSpecialConfig("replicaof", "slaveof", IMMUTABLE_CONFIG | MULTI_ARG_CONFIG, setConfigReplicaOfOption, getConfigReplicaOfOption, rewriteConfigReplicaOfOption, NULL),
     createSpecialConfig("latency-tracking-info-percentiles", NULL, MODIFIABLE_CONFIG | MULTI_ARG_CONFIG, setConfigLatencyTrackingInfoPercentilesOutputOption, getConfigLatencyTrackingInfoPercentilesOutputOption, rewriteConfigLatencyTrackingInfoPercentilesOutputOption, NULL),
 
+    /* Capabalities */
+    createBoolConfig("info-simple-for-sentinel", NULL, IMMUTABLE_CONFIG, server.info_simple_for_sentinel, 1, NULL, NULL),
```

Comment:
Is this config only for sentinel? If yes, I think it should be moved to sentinel.c,  we have some specific config parameters for sentinel node.

Reply:
yes，this config only for sentinel, but it`s mean valkey has a certain ability that sentinel could send diff command to collect instance stats.
It`s a server capability. I think it can`t move to sentinel.c.